docs/sources/migration-guide/migrating-from-single-zone-with-helm.md

```diff
@@ -0,0 +1,719 @@
+
+   1. Wait until queriers and rulers have restarted and are ready.
+
+   1. Monitor resource utilization of queriers and rulers and scale up if necessary. Turning off shuffle sharding may increase resource utilization.
+
+1. Enable zone-awareness on the write path.
+
+   Replace the contents of the `migrate.yaml` file with:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         writePath: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step3"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   In this step the flag `-ingester.ring.zone-awareness-enabled=false` is removed from distributors and rulers.
+
+1. Once all distributors and rulers have restarted and are ready, wait 12 hours.
+
+   The 12 hours is calculated from the `querier.query_store_after` Grafana Mimir parameter.
+
+1. Enable zone-awareness on the read path.
+
+   Replace the contents of the `migrate.yaml` file with:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         writePath: true
+         readPath: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step4"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   In this step the flag `-ingester.ring.zone-awareness-enabled=false` is removed from queriers.
+
+1. Wait until all queriers have restarted and are ready.
+
+1. Exclude non zone-aware ingesters from the write path.
+
+   Replace the contents of the `migrate.yaml` file with:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         writePath: true
+         readPath: true
+         excludeDefaultZone: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step5"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   In this step the flag `-ingester.ring.excluded-zones=zone-default` is added to distributors and rulers.
+
+1. Wait until all distributors and rulers have restarted and are ready.
+
+1. Scale down non zone-aware ingesters to 0.
+
+   Replace the contents of the `migrate.yaml` file with:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         writePath: true
+         readPath: true
+         excludeDefaultZone: true
+         scaleDownDefaultZone: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step6"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+1. Wait until all non zone-aware ingesters are terminated.
+
+1. Delete the default zone.
```

Comment:
This felt a but cryptic. You are telling me to get rid of migrate.yaml and add parts of it back to my custom.yaml. But the step is called "delete the default zone." The step also changes some CLI flags and removes some annotations on services. Maybe add more detail for what is happening

Reply:
Different thing: we don't tell them to clean up the PVs after this is deleted. I think it's a friendly thing to do - "heads up, if you don't delete the PVs you will keep paying for them." Same goes for AM and st-gw